src/v2/guide/single-file-components.md

```diff
@@ -1,43 +1,45 @@
-- **No CSS support** means that while HTML and JavaScript are modularized into components, CSS is conspicuously left out
-- **No build step** restricts us to HTML and ES5 JavaScript, rather than preprocessors like Pug (formerly Jade) and Babel
+- **Les définitions globales** forcent à avoir un nom unique pour chaque composant
+- **Les templates sous forme de chaîne de caractères** ne bénéficient pas de la coloration syntaxique et requiert l'usage de slashes disgracieux pour le HTML multilignes.
+- **L'absence de support pour le css** signifie que le css ne peut pas être modularisé comme HTML et Javascript
+- **L'absence d'étape de build** nous restreint au HTML et à Javascript ES5, sans pouvoir utiliser des preprocesseurs tels que Babel ou Pug (précédemment Jade).
 
-All of these are solved by **single-file components** with a `.vue` extension, made possible with build tools such as Webpack or Browserify.
 
-Here's a simple example of a file we'll call `Hello.vue`:
+Tous ces désavantages sont résolus par les composants monofichier avec une extension `.vue`, rendus possibles par les outils de build tels que Webpack ou Browserify.
+
+Voici un exemple simple de fichier que nous appellerons `Hello.vue` :
 
 <img src="/images/vue-component.png" style="display: block; margin: 30px auto">
 
-Now we get:
+Maintenant nous avons :
 
-- [Complete syntax highlighting](https://github.com/vuejs/awesome-vue#syntax-highlighting)
-- [CommonJS modules](https://webpack.github.io/docs/commonjs.html)
-- [Component-scoped CSS](https://github.com/vuejs/vue-loader/blob/master/docs/en/features/scoped-css.md)
+- [Une coloration syntaxique complète](https://github.com/vuejs/awesome-vue#syntax-highlighting)
+- [Des modules CommonJS](https://webpack.github.io/docs/commonjs.html)
+- [Du css à la portée limitée au composant](https://github.com/vuejs/vue-loader/blob/master/docs/en/features/scoped-css.md)
 
-As promised, we can also use preprocessors such as Jade, Babel (with ES2015 modules), and Stylus for cleaner and more feature-rich components.
+Et comme promis, nous pouvons aussi utiliser des preprocesseurs tels que Jade, Babel (avec les modules ES2015), et Stylus pour obtenir des composants plus lisibles et plus riches en fonctionnalités.
 
 <img src="/images/vue-component-with-preprocessors.png" style="display: block; margin: 30px auto">
 
-These specific languages are just examples. You could just as easily use Bublé, TypeScript, SCSS, PostCSS - or whatever other preprocessors that help you be productive. If using Webpack with `vue-loader`, it also has first-class support for CSS Modules.
+Ces languages spécifiques ne sont que des exemples; vous pourriez tout aussi aisément utiliser Bublé, Typescript, SCSS, PostCSS - ou tout autre preprocesseur qui vous aide à être productif. Si vous utilisez Webpack avec `vue-loader`, vous aurez aussi un outil de premier choix pour les modules CSS.
+
+### Qu'en est-il de la séparation des responsabilités ?
```

Comment:
a priori on le traduit plus souvent en "sépération des préoccupations" (ref: http://weblog.godlike.fr/2012/06/06/abap-bonnes-pratiques-separation-of-concerns/), mais le terme anglais est celui qui dispose du plus de branding et donc de reconnaissance. Je suis OK pour laisser le mot anglais ou l'indiquer entre parenthèses derrière.